delivery_carrier_label_default/models/delivery_carrier.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2012 Akretion <http://www.akretion.com>.
+# Copyright 2013-2016 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = "delivery.carrier"
+
+    def alternative_send_shipping(self, pickings):
+        default_label = pickings.generate_default_label()
```

Comment:
Here we call `generate_default_label` method  with pickings in plural. However, inside `generate_default_label` method we have a` self.ensure_one() `. This is kind of misleading or there is a culprit :thinking:

Reply:
You are totally right!
I'll recheck and fix. This code was already present in the 14.0 PR
https://github.com/OCA/delivery-carrier/pull/456/files#diff-3255e5bb12a1416415d6529746f8f0f77c07f3ba798e88d57509aa07684b9a72

Reply:
I'm trying to understand the needs of this method here.
It is mentioned here: https://github.com/OCA/delivery-carrier/pull/456#issuecomment-1316635156

Do you have any context about this ?

ping @sebalix

Reply:
I decided to remove this relation because I'm not sure that's linked to this module.